app/context/ceramicContext.tsx

```diff
@@ -393,6 +434,21 @@ export const CeramicContextProvider = ({ children }: { children: any }) => {
     }
   }, [ceramicClient]);
 
+  useEffect(() => {
+    if (pdbPass && ceramicClient) {
```

Comment:
Since cdbStamps is initialized as an empty array, if pdbPass is loaded before ComposeDatabse.getPassport() is called/completed, then it'll try to write all the stamps to Ceramic.

The easiest thing to do is probably to change it back to similar to how it was in the original PR.

Otherwise, I think it would work if cdbStamps were initialized to undefined and then checked in the useEffect? If you go this route, could you rename cdbStamps and pdbPass to something more descriptive? Maybe initialCeramicStamps and initialPassport?

Reply:
Thanks @lucianHymer  - I went with the latter (since the original PR was disrupting some of the original tests). Handled in commit [1d17930](https://github.com/gitcoinco/passport/pull/2401/commits/1d1793099b4668189a8ed2d340fe7c458c5275ee)